index.html

```diff
@@ -191,10 +240,16 @@ <h4 > LET’S CONNECT</h4>
                 <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
                 <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
             </ul>
+            <ul class="Programs-desclist">
+                <li   href="#"><img class="Programs-desck" src="./image/descktop/Frame 10 (1).png" alt=""></li>
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 11 (1).png" alt=""></li>
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 9 (2).png" alt=""></li>
+            </ul>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
         </ul>
     </div>
+</div>
 </section>
 <section class="cardcontact">
```

Comment:
Kindly fix the gray area and try to follow the Figma template.
Figma template:
<img width="564" alt="Screenshot 2022-12-16 at 10 17 36 PM" src="https://user-images.githubusercontent.com/25887253/208172585-f3510e8a-7910-4668-83a6-40ca6cdaa981.png">
Your design:
<img width="1440" alt="Screenshot 2022-12-16 at 10 00 54 PM" src="https://user-images.githubusercontent.com/25887253/208172974-a51f77e4-6ab7-4bcb-8266-80ee2bd2d445.png">